phpunit

```diff
@@ -1,38 +1,12 @@
 #!/usr/bin/env php
 <?php
-/* PHPUnit
+/*
+ * This file is part of the PHPUnit.
```

Comment:
Surely `This file is part of PHPUnit.` rather than `This file is part of the PHPUnit.`?